eng/pipelines/runtime.yml

```diff
@@ -756,9 +752,23 @@ jobs:
       jobTemplate: /eng/pipelines/libraries/build-job.yml
       buildConfig: Debug
       platforms:
-        - Linux_arm
-        - Linux_musl_arm
-        - Linux_musl_arm64
```

Comment:
These three debug builds are no longer needed: we no longer run Linux_arm in PRs (replaced by Linux_arm64) and Linux_musl_arm/Linux_musl_arm64 now run in release. However, windows-x86 is moved here (see comment above).